index.js

```diff
@@ -1,4 +1,7 @@
-var defaultIsMergeableObject = require('is-mergeable-object')
+var isPlainObj = require('is-plain-obj')
+function defaultIsMergeableObject(value) {
```

Comment:
mmm, now I'm thinking that this should change to `defaultIsMergeable`, and the option should change from `isMergeableObject` to `isMergeable`, since we're really just checking if something is mergable – with custom merge functions, people can merge whatever they want, and our default implementation intentionally includes arrays as well as "regular" objects.

Reply:
Should there be backward compatibility for `opts.isMergeableObject`?

Reply:
hmm, as long as we can do it in <20 characters I'll say yeah :-x